bin/check-changelog.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * Given a set of file paths as arguments, checks whether the file paths also
+ * include a requisite CHANGELOG entry. The file paths should include packages
+ * maintained with a CHANGELOG at, e.g. `packages/i18n/CHANGELOG.md`.
+ *
+ * Example (Error):
+ *
+ * ```
+ * node bin/check-changelog.js /Gutenberg/packages/i18n/src/index.js
```

Comment:
Is this comment correct? I would expect something containing `Missing CHANGELOG.md entry for...`

Reply:
> Is this comment correct? I would expect something containing `Missing CHANGELOG.md entry for...`

It's an example of a failing call, so yes, it would log the message.